packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -0,0 +1,248 @@
+/**
+ * @Prop   --cbp-tooltip-color: var(--cbp-color-text-lightest);
+ * @Prop   --cbp-tooltip-color-dark: var(--cbp-color-text-darkest);
+ * @Prop   --cbp-tooltip-color-bg: var(--cbp-color-gray-cool-80);
+ * @Prop   --cbp-tooltip-color-bg-dark: var(--cbp-color-gray-cool-5);
+ * @Prop   --cbp-tooltip-control-focus: 1px solid red;
+ * @Prop   --cbp-tooltip-control-focus-dark: 1px solid cyan;
+ * @Prop   --cbp-tooltip-padding: var(--cbp-space-inset-4x, 1rem);
+ * @Prop   --cbp-tooltip-gap: 0px;
+ * @Prop   --cbp-tooltip-offset: 0px;
+ * @Prop   --cbp-tooltip-link-decoration: none;
+ * @Prop   --cbp-tooltip-caret-size: 15px;
+ * @Prop   --cbp-tooltip-caret-offset: calc(var(--cbp-tooltip-caret-size) / 2);
+ * @Prop   --cbp-tooltip-color-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-tooltip-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-tooltip-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-tooltip-color-bg-hover-dark: var(--cbp-color-interactive-disabled-light);
+ * @Prop   --cbp-tooltip-color-focus: var(--cbp-color-text-lightest);
+ * @Prop   --cbp-tooltip-color-focus-dark: var(--cbp-color-text-darkest);
+ * @Prop   --cbp-tooltip-color-bg-focus: var(--cbp-color-interactive-focus-dark);
+ * @Prop   --cbp-tooltip-color-bg-focus-dark: var(--cbp-color-interactive-focus-light);
+ */
+ :root{
+ 
```

Comment:
these variables are not clearly named. I think you might want to specify --cbp-tooltip-content and --cbp-tooltip-control to make it clear what they represent.

Reply:
update var names to distinguish between  content & control